pipelines/nextflow/README.md

```diff
@@ -7,6 +7,16 @@ Please, refer to the "Installation" section of the [Nextflow pipelines document]
 Please, refer to the "Running a pipeline" section of the [Nextflow pipelines document](/docs/nextflow.md#running_a_pipeline).
 And to individual README(s) in '/pipelines/nextflow/workflows/' subfolders.
 
+### Trace and log reports
+Add `-with-trace` to your `nextflow run ...` command to activate the generation of a trace file with useful information such as resource usage of each job and their hash directory. More information can be found in the [Nextflow documentation](https://www.nextflow.io/docs/latest/tracing.html).
+
+Most of our jobs in our pipelines have the `--verbose` flag set as default, providing a more detailed log. You can generate a single log report with the information from every job by running:
+```bash
+nextflow log <run name> -t log_template.md > log_report.md
+```
+
+However, sometimes this report may be too large to be useful, so the previous command includes a `-filter` option, e.g. `-filter 'name =~ /DOWNLOAD_ASM_DATA.*/'` will provide a log report only for those jobs whose name starts with `DOWNLOAD_ASM_DATA`.
```

Comment:
A note about this: "[...] this report may be too large to be useful [...]"

This is inevitable if we process a large number of species, but if we look at each genome, we need the INFO logs to be at least readable: i.e. what we write to INFO needs to be relatively short and informative, so should ideally be less than a couple of pages long per process.

(That said if the logs are full of warnings we do need all the details)